content/documents/requesting_funds.md

```diff
@@ -0,0 +1,61 @@
+
+### Substance of Proposal
+- Specific area(s) of development:
+- Length of contract:
+- Wage in USD:
+
+## Conferences and Trade Shows
+Also important to the Project is increasing the community of users, testers and developers. A very effective method of doing this is to attend conferences, trade shows and exhibitions, where people are often exposed to Haiku for the first time and get to see it hands on. In addition to casual attendees, several well known people in the FOSS (Free and Open Source) world get to speak with project representatives and see how Haiku is shaping up. This can often be an opportunity to strengthen our networking among FOSS advocates and other projects. These shows can also be an opportunity to locate commercial interest in Haiku. While many of the current shows Haiku attends occur in the west coast United States, several also occur in the EU, such as RMLL and FOSDEM.
+
+The goals are to provide reimbursement costs for the individuals who promote Haiku (demonstrating, running the booth, giving presentations, ..) and to pay for show expenses (registration fees, booth fees, and related items).
+
+### Substance of Proposal
+- Name of the conference:
+- Conference website:
+- Duration:
+- Fees directly associated with conference
+	- Entrance fee:
+	- Electricity costs:
+	- Insurance:
+	- Other fees and charges:
+- Travel method & costs:
+- Lodging:
+- Parking:
+When attending multi-day conferences, a stipend of $17/day (aka $5, $7, $5) may be requested to assist with nutritional needs. For single day events, you are encouraged to bring food as that is usually the most cost-effective solution.
+
+## Other Purchases
+In order to support conference and trade show presence, marketing supplies is another area that needs occasional funding. Hard goods, such as banners, presentation equipment, and fliers increase the visual appeal and effectiveness of Haiku at trade shows. Many of these items can be purchased and reused for several years, or purchased in bulk and distributed to several individuals attending different shows at different times.
+
+Another group of expenses we would like to officially endorse, is purchasing items used for the development process. Examples would be hardware for writing new drivers or debugging existing ones, technical documents purchased from standards organizations, etc. We hope to also be able to provide developers with replacement hardware in the event of the inevitable failures that occur.
```

Comment:
There is some hardware owned by Haiku inc that should be listed somewhere (either bought for developers, or donations from ahiku users and contributors or other non-profits):
- The Mac Minis from Mozilla
- Some ARM devboards (I think a gumstick currently in the hands of mmu_man, and possibly a beagleboard for a GSoC student, but we'd need to check
- Some VIA x86 CPU based embedded PC board with GPS receiver, Oxford Semi UART chip, etc (currently in my hands, but I can send it to other devs if they are interested)
- Anything else?

I think this deserves a new page and there could be a link here to show these as examples.

Reply:
Hello @pulkomandy,

I've created a page paraphrasing your comment. Feel free to add to it once you've checked what hardware Haiku owns.

Thanks!

Reply:
Some of the mac mini's have been "released" by the Inc. for their years of service, some are missing, I have 4 of them sitting under my desk collecting dust. They didn't actually cost Haiku, Inc. anything, and were a generous donation from Mozilla.  I've been pondering giving them to Haiku community members to develop Haiku on, but honestly a good chunk of them would just immediately go to EBay if I did that :-)

They're Core 2 Duo Mac's and honestly, running 4 of them is a waste of power as of a few years ago.  I took all 4 of mine offline since my personal Dell Core i7 system I have dedicated to nightly builds uses less energy and is faster.

Reply:
@kallisti5 I've added a note after the Mac Minis saying that the Inc. released some to Haiku developers.